avocado/core/extension_manager.py

```diff
@@ -44,6 +54,13 @@ def __init__(self, name, entry_point, plugin, obj):
         self.plugin = plugin
         self.obj = obj
 
+    def __lt__(self, other):
```

Comment:
If you are overriding `__lt__` it would be nice to do the same with __le__ or __gt__ and __ge__.

Reply:
I was expecting that the `Extension` will be only necessary to sort. But you are right that with this change, the comparison will be inconsistent. So I also updated the others.